app/jobs/set_content_type_job.rb

```diff
@@ -41,7 +42,7 @@ def set_content_type(cocina_object, success, failure)
 
   def cocina_update_attributes(cocina_object)
     {}.tap do |attributes|
-      attributes[:type] = Constants::CONTENT_TYPES[@new_content_type]
+      attributes[:type] = @new_content_type
```

Comment:
this and other similar changes are because the drop down menus now have the actual cocina types (e.g. `https://cocina.sul.stanford.edu/models/resources/page`) instead of the display value (e.g. `page`) -- so we don't look them up anymore but use them directly (as is done in https://github.com/sul-dlss/argo/blob/main/app/forms/content_type_form.rb)